conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
Otherwise how would that be implemented? Just always `ceil()` it to the next multiple of the raw interval? I guess that would work too